devito/arch/compiler.py

```diff
@@ -671,37 +681,28 @@ def __lookup_cmds__(self):
 class IntelCompiler(Compiler):
 
     def __init__(self, *args, **kwargs):
-        super(IntelCompiler, self).__init__(*args, **kwargs)
-
-        self.cflags.append("-xhost")
+        super().__init__(*args, **kwargs)
 
-        language = kwargs.pop('language', configuration['language'])
+        self.cflags.append("-xHost")
```

Comment:
ideally this line goes after pulling platform and language, but we can address it in another PR if it can spare a useless CI spin

Reply:
ok